readme.md

```diff
@@ -58,8 +58,8 @@ just type `readTime` instead of `this.state.readTime`. Saves a **lot** of
 typing.
 
 The next thing we should do is change our variable assignment from a `let`
-to a `const`. This just ensures that our variables will not change. Anything
```

Comment:
`const` does not ensure immutability. It prevents reassignment, but does nothing to prevent changing the values of existing assignment.

eg ```
// fine
const foo = { qux: 44 }
foo.qux++

// explodey
const fooCount = 44
fooCount++